python/xorbits/deploy/kubernetes/client.py

```diff
@@ -143,6 +146,8 @@ def __init__(
         self._cluster_type = self._get_cluster_type(cluster_type)
         self._ingress_name = "xorbits-ingress"
         self._use_local_image = kwargs.pop("use_local_image", False)
+        self._external_storage = external_storage
```

Comment:
You should check the ``external_storage`` input. Currently we only support ``juicefs``. If ``external_storage`` is not ``None`` or ``juicefs``, just raise an error.